src/components/GooglePay/types.ts

```diff
@@ -24,6 +28,7 @@ export interface GooglePayPropsConfiguration {
 export interface GooglePayProps extends UIElementProps {
     environment?: google.payments.api.Environment | string;
     configuration?: GooglePayPropsConfiguration;
+    paymentMethods?: PaymentMethod[];
```

Comment:
Is there any reason why we added `paymentMethods` into the `GooglePayProps` types?

Reply:
This can go - they were propagated in an earlier commit that was since revised